### ISSUE TITLE
Stack Loss Calculator

### ISSUE THREAD

**Issue by koay9f** [ORNL-AMO/AMO-Tools-Desktop#5250]
Need to update the stack loss calculator to match the flue gas calculator.
I think we just need to add Fuel Temp to Gas fuel type. Don't think it needs anything else.

Update standalone and modal.

**Comment by koay9f** [comment]
Okay, apparently I didn't pay enough attention and we have 2 different fields titled "Ambient Air Temperature"
I think I remember we decided to set "Combustion Air Temp" and "Fuel Temp" equal and ask 1 field (Amibient Air), but then we added ambient temp to the calculations.

Two possible solutions, I'll let you decide which we have time for
1 - remove one of the ambient air temp fields and set all three variables passed to the suite (ambient, fuel, combustion air temps) equal and just have one field (default = 60 F)
2 - make a accoridian? window with ambient air, comubstion air, and fuel temps AND "Mositure in combustion air" all in there. Title window: "Detailed Analysis". Set defaults to 60 F, 60F, 60F, 0% respectivly. 
(60F is 15.6C for the metric examples/defaults
